nebd/src/part2/file_entity.cpp

```diff
@@ -361,6 +363,11 @@ int NebdFileEntity::UpdateFileStatus(NebdFileInstancePtr fileInstance) {
     fileInstance_ = fileInstance;
     status_ = NebdFileStatus::OPENED;
     timeStamp_ = TimeUtility::GetTimeofDayMs();
+    OpenFlags flags;
+    if (fileInstance->xattr.count(kOpenFlagsAttrKey) &&
+        flags.ParseFromString(fileInstance->xattr.at(kOpenFlagsAttrKey))) {
+        openFlags_.reset(new OpenFlags{flags});
+    }
```

Comment:
it seems this code duplicates with L121-L125

Reply:
> it seems this code duplicates with L121-L125

fix